configuration.go

```diff
@@ -334,6 +334,7 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultSidecar.Endpoint = "http://127.0.0.1:7777"
 	defaultSidecar.Frontend = "sidecar.toml"
 	defaultSidecar.RefreshConn = flaeg.Duration(60 * time.Second)
+	defaultSidecar.MaxConns = 300
```

Comment:
This is a _super_ low default. HAproxy defaults to 2000 for example:

>Also, when <conns> is set to large values, it is possible that the servers
are not sized to accept such loads, and for this reason it is generally wise
to assign them some reasonable connection limits.

> By default, this value is set to 2000.